setup.cfg

```diff
@@ -0,0 +1,8 @@
+[tool:pytest]
+testpaths=gatorgrouper/tests
+
+[mutmut]
+paths_to_mutate=gatorgrouper/
+backup=False
+tests_dir=tests
+runner=python -m pytest
```

Comment:
Should this mention pipenv? This may be why many mutants are surviving.

Reply:
@leonardoz15 if this can be fixed or you can confirm it is not a problem, this can be merged!